node/service/src/block_sealing.rs

```diff
@@ -92,10 +92,16 @@ pub fn start_frequency_dev_sealing_node(
 				network_provider: Arc::new(network.clone()),
 				enable_http_requests: true,
 				custom_extensions: move |_hash| {
-					let cloned = rpc_address.clone();
-					vec![Box::new(OcwCustomExt(cloned)) as Box<_>]
+					rpc_addresses
+						.iter()
+						.map(|a| {
+							let addr_cloned = a.clone();
+							Box::new(OcwCustomExt(addr_cloned)) as Box<_>
+						})
+						.collect()
```

Comment:
This doesn't look correct to me. I don't think we would like to create a new extension for each rpc address

Reply:
We will select the first address and create the one extension.